src/watertap_contrib/reflo/property_models/fo_draw_solution_properties.py

```diff
@@ -0,0 +1,815 @@
+        )
+        self.osm_coeff_param_3 = Param(
+            initialize=5.75176e7,
+            units=pyunits.Pa,
+            doc="Osmotic coefficient parameter 3",
+        )
+        # specific heat parameters, derived from experimental data
+        cp_units = pyunits.J / (pyunits.kg * pyunits.K)
+        self.cp_phase_param_A0 = Param(
+            initialize=4.20003e3,
+            units=cp_units,
+            doc="Specific heat of seawater parameter A0",
+        )
+        self.cp_phase_param_A1 = Param(
+            initialize=-3.49888e3,
+            units=cp_units,
+            doc="Specific heat of seawater parameter A1",
+        )
+        self.cp_phase_param_A2 = Param(
+            initialize=1.33883e3,
+            units=cp_units,
+            doc="Specific heat of seawater parameter A2",
+        )
+        self.separation_heat = Param(
+            initialize=105,
+            units=pyunits.MJ / pyunits.m**3,
+            doc="Separation heat per m3 of the separated water",
+        )
+        # ---------------------------------------------------------------------
+        # Set default scaling factors
+        self.set_default_scaling("temperature", 1e-2)
+        self.set_default_scaling("pressure", 1e-6)
+        self.set_default_scaling("dens_mass_phase", 1e-3, index="Liq")
+        self.set_default_scaling("cp_mass_phase", 1e-3, index="Liq")
+        self.set_default_scaling("enth_mass_phase", 1e-5, index="Liq")
+        self.set_default_scaling("heat_separation_phase", 1e-5, index="Liq")
+        self.set_default_scaling("liquid_separation", 1, index="Liq")
+
+    @classmethod
+    def define_metadata(cls, obj):
+        obj.add_default_units(
+            {
+                "time": pyunits.s,
+                "length": pyunits.m,
+                "mass": pyunits.kg,
+                "amount": pyunits.mol,
+                "temperature": pyunits.K,
+            }
+        )  # Not used
+
+        obj.add_properties(
+            {
+                "flow_mass_phase_comp": {"method": None},
+                "temperature": {"method": None},
+                "pressure": {"method": None},
+                "flow_vol_phase": {"method": "_flow_vol_phase"},
+                "conc_mass_phase_comp": {"method": "_conc_mass_phase_comp"},
+                "mass_frac_phase_comp": {"method": "_mass_frac_phase_comp"},
+                "dens_mass_phase": {"method": "_dens_mass_phase"},
+                "pressure_osm_phase": {"method": "_pressure_osm_phase"},
+                "cp_mass_phase": {"method": "_cp_mass_phase"},
+                "enth_mass_phase": {"method": "_enth_mass_phase"},
+                "heat_separation_phase": {"method": "_heat_separation_phase"},
+                "enth_flow": {"method": "_enth_flow"},
+            }
+        )
+
+        obj.define_custom_properties(
+            {
+                # "heat_separation_phase": {"method": "_heat_separation_phase"},
+            }
+        )
```

Comment:
Can be removed?

Reply:
Yes, it should be removed, as this property is already in the "normal" property list above it.